plugins/app_plugin.go

```diff
@@ -110,12 +117,23 @@ func (p CPUEntitlementPlugin) Run(cli plugin.CliConnection, args []string) {
 }
 
 func (p CPUEntitlementPlugin) GetMetadata() plugin.PluginMetadata {
+	major := 0
+	minor := 0
+	build := 0
+	if p.Version != "" {
+		version := strings.Split(p.Version, ".")
+		if len(version) == 3 {
+			major, _ = strconv.Atoi(version[0])
+			minor, _ = strconv.Atoi(version[1])
+			build, _ = strconv.Atoi(version[2])
+		}
+	}
 	return plugin.PluginMetadata{
 		Name: "CPUEntitlementPlugin",
 		Version: plugin.VersionType{
-			Major: 0,
-			Minor: 0,
-			Build: 2,
```

Comment:
Wow, was the plugin version hard-coded to 0.0.2 before?

Reply:
Yes, unfortunately.